content/chainguard/chainguard-images/authenticating.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Authenticating to Chainguard Registry"
+type: "article"
+description: "A guide on authenticating to the Chainguard Registry to get images"
+date: 2023-03-21T15:10:16+00:00
+lastmod: 2023-03-21T15:10:16+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 500
+toc: true
+---
+
+### Public Images
+
+All public Chainguard Images are available for free.
+
+However, logging in with a Chainguard account and authenticating when pulling from the registry can provide a mechanism for Chainguard to contact you if there are any issues with images you are pulling.
+
+This may include upcoming deprecations or changes in behavior, or critical vulnerabilities found and remediations applied to images being pulled.
```

Comment:
I don't love the cadence of this sentence but I'm not awake enough to wordsmith it yet.